src/js/components/Modal.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import _ from 'lodash';
+import {TextAreaAndButton} from './GeneralComponents';
+import {ReplyToTweet} from '../services/httpCall';
+import {Tweet} from './Tweets';
+
+const e = React.createElement;
+
+class Modal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            modalMsgLength: 0,
+            returnMessage: null,
+            retAppend: "",
+            textBox: null,
+        }
+
+        this.updateReturnMessage = this.updateReturnMessage.bind(this);
+        this.clearTextBox = this.clearTextBox.bind(this);
+    }
+
+    componentDidMount() {
+        let tbClass = document.getElementsByClassName("modalTextArea");
+        this.setState({
+            textBox: tbClass && tbClass[0]
+        })
+    }
+
+    updateUI() {
+        if(this.state.textBox && this.state.textBox.value) {
```

Comment:
can be simplified to the following i believe
this.setState({
modalMsgLength: this.state.textBox && this.state.textBox.value && this.state.textBox.length || 0